commands/commands.go

```diff
@@ -62,6 +62,17 @@ var (
 		Usage: "Print a report instead of csv",
 	}
 
+	limitFlag = cli.IntFlag{
+		Name:  "limit, li",
+		Usage: "Limit the outputs of the result to `LIMIT` values",
+		Value: 1000,
+	}
+
+	noLimitFlag = cli.BoolFlag{
```

Comment:
So random thought check with Ethan, but mongo implements a limit flag with an option of 0. If the limit is equal to zero that indicates no limits. Would it be easier to use that or keep as is. Also can we default the value to False?

Reply:
Ethan just mentioned [in the issue](https://github.com/activecm/rita/issues/466) that 1000 as a default was a requirement for the pull, it shouldn't be any issue to change it though

Reply:
I wasn't super clear but 1000 is good.
If a user tries something like `--li 0` they get an error saying the limit must be positive. If we changed the PR so the --nl flag doesn't exist and we make `--li 0` be the no limit flag if that makes sense

Reply:
It's either way, I'll defer to @ethack 's wishes on it, the no limit flag was just another parameter request in the issue

Reply:
Sounds good! We'll leave it in then

Reply:
I'm good either way. The `--no-limit` flag was given as an option equivalent to `--limit 0`. I left it to whoever implemented it to weigh the pros and cons. Since Mongo doesn't allow passing a straight 0 into the limit field in aggregations we'd still have to special case the 0 in code like we do now with the `if !noLimit` checks. So it wouldn't simplify the code at least. But the user experience I'm not sure of which way is better. I'll merge it as-is since this was resolved, but if there's a strong preference either way before we do a new rita release we can change it later.